src/Validation/validateInput.js

```diff
@@ -0,0 +1,10 @@
+export default function validateInput(inputMail, inputPassword) {
+  var mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  if (inputMail == '' || mailformat.test(inputMail) == false) {
+    alert('Please provide valid email');
+  } else if (inputPassword == '') {
```

Comment:
Are we not applying any validation on the Password? like min-length or characters allowed validations.

Reply:
Okay I will add validation on password.